org.dupot.littleadventure.yml

```diff
@@ -0,0 +1,30 @@
+id: org.dupot.littleadventure
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+command: /app/bin/littleAdventure.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+modules:
+  - name: littleAdventure
+    buildsystem: simple
+    build-commands:
+      - install -Dm555 littleAdventure.x86_64 /app/bin/littleAdventure.x86_64
+      - install -Dm644 littleAdventure.x86_64 /app/bin/littleAdventure.pck
+      - install -Dm644 512x512.png /app/share/icons/hicolor/512x512/apps/org.dupot.littleadventure.png
+      - install -Dm644 org.dupot.littleadventure.desktop /app/share/applications/org.dupot.littleadventure.desktop
+      - install -Dm644 org.dupot.littleadventure.appdata.xml /app/share/metainfo/org.dupot.littleadventure.appdata.xml
+    sources:
+      - type: file
+        path: littleAdventure.pck
+      - type: file
+        path: littleAdventure.x86_64
```

Comment:
definitely not.

Reply:
this repository should NOT contain binaries or the upstream source code

Reply:
> 

Hi, i create a new branch with the correct name: org.dupot in place of net.dupot, and i put all files in commit, because i don't understand why the sha256 check is different than mine generated on my computer

Reply:
> 

ok i will remove it, but i don't understand why the sha256 is different
For information, i try on my computer to build to check, and each time i launch, i have on flatpak-build a different sha256 sum..

Where can i ask support about that ?

my exple (you can see that checksum is different each time i launch flatpak-builder..):
$ flatpak-builder build-dir org.dupot.littleadventure.yml --force-clean
Emptying app dir 'build-dir'
Downloading sources
Downloading https://github.com/imikado/dupotLittleAdventure/blob/main/export/linux/littleAdventure.pck
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  125k    0  125k    0     0   240k      0 --:--:-- --:--:-- --:--:--  240k
Failed to download sources: module littleAdventure: Wrong sha256 checksum for littleAdventure.pck, expected "2f95d123f6a4cab0c9f8df45932fae43dcc3139b4aec547b34bbfa60dd4a4537", was "bc39ed72e605ef3a7ce65379083103cd99b4d79b65c2a09723b66d24fc2c2977"
╭─mika@mika-Precision-3541 in ~/code/github/flatpak/flathub on org.dupot.littleadventure ✘ 
╰$ flatpak-builder build-dir org.dupot.littleadventure.yml --force-clean
Downloading sources
Downloading https://github.com/imikado/dupotLittleAdventure/blob/main/export/linux/littleAdventure.pck
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  125k    0  125k    0     0   898k      0 --:--:-- --:--:-- --:--:--  898k
Failed to download sources: module littleAdventure: Wrong sha256 checksum for littleAdventure.pck, expected "2f95d123f6a4cab0c9f8df45932fae43dcc3139b4aec547b34bbfa60dd4a4537", was "9e747344978dcccf5ebfdfb7a15c69d0aad3b81b8d3150a62bd1e56c7e4f1f64"